tests/resources/sips/newspaper/newspaper.sip.shacl.ttl

```diff
@@ -98,6 +111,16 @@
         sh:maxCount 1 ;
     ],
     # DateIssued?
+    [
+        a sh:PropertyShape ;
+        sh:path dct:issued ;
+        sh:nodeKind sh:Literal ;
+        sh:datatype edtf:EDTF-level1 ;
+        sh:name "Date issued"@en ;
+        sh:description "The date of issue of the newspaper."@en ;
+        sh:minCount 1 ;
+        sh:maxCount 1 ;
+    ],
```

Comment:
`dct:issued` als alternatief van mapping, ook volgens: https://samvera.atlassian.net/wiki/spaces/samvera/pages/405212209/Previous+MODS+to+RDF+mappings+reflection+and+feedback
Dan ook beter `mods:dateCreated` omzetten naar `dct:created`?

Reply:
@lennertvandevelde misschien is het interessant om deze mapping volledig te volgen ipv via MODSRDF te gaan?